dev/com.ibm.ws.io.smallrye.reactive.messaging/src/com/ibm/ws/microprofile/reactive/messaging/OLReactiveMessagingExtension.java

```diff
@@ -29,9 +29,9 @@
 import io.smallrye.reactive.messaging.extension.MediatorManager;
 import io.smallrye.reactive.messaging.extension.ReactiveMessagingExtension;
 import io.smallrye.reactive.messaging.extension.StreamProducer;
-import io.smallrye.reactive.messaging.impl.ConfiguredStreamFactory;
-import io.smallrye.reactive.messaging.impl.InternalStreamRegistry;
-import io.smallrye.reactive.messaging.impl.StreamFactoryImpl;
+import io.smallrye.reactive.messaging.impl.ConfiguredChannelFactory;
+import io.smallrye.reactive.messaging.impl.InternalChannelRegistry;
+import io.smallrye.reactive.messaging.impl.LegacyConfiguredChannelFactory;
 
 @Component(service = WebSphereCDIExtension.class, configurationPolicy = ConfigurationPolicy.IGNORE, property = { "service.vendor=IBM" })
```

Comment:
Need to add property `"application.bdas.visible=true"`

Reply:
done